.github/workflows/book.yml

```
@@ -52,19 +52,19 @@ jobs:
         path: book_source/_book/
     # download documentation repo
     - name: Checkout documentation repo
-      if: github.event_name != 'pull_request'
+      if: github.event_name == 'push' && (startsWith(github.ref, 'refs/tags/') || github.ref == 'refs/heads/master' || github.ref == 'refs/heads/develop')
```

do we need this complexity. I think we can just check to see if it is push, since the only reason we run this workflow is push on (master/dev and tag).
```suggestion
      if: github.event_name == 'push'
```

---

Yeah sure. I just added this in case it would be required for future conditions. I'll change it right away.

---

@robkooper Good catch! Does this apply to line 63 in the deploy step as well?